smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/model/ShapeExt.kt

```diff
@@ -103,6 +103,10 @@ fun MemberShape.needsDefaultValueCheck(model: Model, symbolProvider: SymbolProvi
     return isPrimitiveShape && isNotBoxed && defaultValueNotNull
 }
 
+fun MemberShape.isRequired(): Boolean {
+    return (this.hasTrait<RequiredTrait>())
+}
```

Comment:
can be?
```suggestion
fun Shape.isRequired(): Boolean {
    return (this.hasTrait<RequiredTrait>())
}
```

Reply:
Hmm i don't know if it makes sense to put this on shape, since having the required trait means the shape is a member of some shape

Reply:
https://awslabs.github.io/smithy/1.0/spec/core/constraint-traits.html#required-trait

makes sense, it is member trait only.